src/ast/mod.rs

```diff
@@ -793,28 +777,27 @@ impl fmt::Display for Statement {
                 local,
                 variable,
                 value,
-            } => {
-                f.write_str("SET ")?;
-                if *local {
-                    f.write_str("LOCAL ")?;
-                }
-                write!(f, "{} = {}", variable, value)
-            }
+            } => write!(
+                f,
+                "SET{} {} = {}",
```

Comment:
I think it makes sense to use named arguments here `{local}` etc as well

Reply:
Right. I was too focussed on the other inconsistencies that I missed how I was introducing another one myself. Thanks for catching that!